### PR TITLE
feat: added support for apple m1 using mps as default device

### PR DESCRIPTION
**Pull request by sabrimansor**
added a Fix for issue #231

### CODE REVIEW

**Comment by TylerYep** [comment]
Thanks for the PR. What version did PyTorch start supporting mps as a device? I can use this for testing specific version support.

**Comment by sabrimansor** [comment]
Pytorch 1.13

File: torchinfo/torchinfo.py [hunk 1]
```diff
@@ -457,6 +457,8 @@ def get_device(model: nn.Module) -> torch.device | str:
 
     if model_parameter is not None and model_parameter.is_cuda:
         return model_parameter.device
+    if hasattr(torch.backends, "mps") and torch.backends.mps.is_available():
+        return "mps"
```
Comment (TylerYep) [inline torchinfo/torchinfo.py:461]: Shouldn't this be `torch.device("mps")`?

**Comment by TylerYep** [comment]
Hi, I ran our test suite with `mps` enabled and I don't think it's safe to always use `mps`. Elaborated more in the original issue #231 . Thanks for the PR, but closing for now.

**Pull request closed without merging**